integration/query_bitwise_test.go

```diff
@@ -50,9 +50,11 @@ func TestQueryBitwiseAllClear(t *testing.T) {
 		"Array": {
 			value: primitive.A{1, 5},
 			expectedIDs: []any{
```

Comment:
These tests cannot be moved to compat because `BitData` is not supported. https://github.com/FerretDB/FerretDB/issues/508

Reply:
Let's add a todo about it?
Something like `TODO Move to compat tests when https://github.com/FerretDB/FerretDB/issues/508 is done`.

Reply:
Updated the the issue #508 to move to compat tests by searching for the issue number :+1: